### PR TITLE
[qa] Added commit message style check

### PR DESCRIPTION
**Pull request by marfgold1**
Added commit message style check

### CODE REVIEW

**Comment by marfgold1** [comment]
I'm unsure, where should i put `pip install https://github.com/openwisp/openwisp-utils/tarball/master` ?
Oh and about the commit, sorry for the inconvenience. I will squash them

File: requirements-test.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+openwisp-utils[qa]>=0.2.1
```
Comment (nemesisdesign) [inline requirements-test.txt]: this is not needed

Comment (ppabcd) [reply requirements-test.txt]: hmm installation with travis without requirements-test.txt files

**Comment by ppabcd** [comment]
Remove requirements-test.txt and start install openwisp-utils using travis without file text.

**Comment by ppabcd** [comment]
Don't forget to squash your commit if all done :)

**Comment by marfgold1** [comment]
Is it enough?

> Remove requirements-test.txt and start install openwisp-utils using travis without file text.
> 
> Add `pip install https://github.com/openwisp/openwisp-utils/tarball/master` in `.travis.yml`

Yeah sure, I accidentally delete all commits and made commit c026b56 to be in here :( (my fault)
But it will be fine now, i guess. Sorry, my bad..

File: .travis.yml [hunk 2]
```diff
@@ -36,6 +36,16 @@ script:
   - export cleanup=false
   # run ci tests
   - ${PWD}/tests/runtests.sh
+  - pip install --upgrade pip
```
Comment (nemesisdesign) [inline .travis.yml]: remove this line

Comment (marfgold1) [reply .travis.yml]: I already did that before and the build is still failed.
Maybe I'll try it later.

**Comment by marfgold1** [comment]
Still failed, any help? @nemesisdesign

File: .travis.yml (hunk 2 above)
Comment (ppabcd) [reply .travis.yml]: hmm i see this travis using docker. Maybe @hispanico can help you.

**Comment by marfgold1** [comment]
@dwang ok next time for sure i'll try it

**Comment by marfgold1** [comment]
Oh yeah also thanks to @dwang (I already say thanks in conversation but forgot to did it in here, sorry)
And thanks @nemesisdesign :D

**Pull request merged**